easy-ui-react/package.json

```diff
@@ -41,8 +41,10 @@
     "overlayscrollbars": "^2.2.1",
     "overlayscrollbars-react": "^0.5.1",
     "react-aria": "^3.26.0",
+    "react-is": "^18.2.0",
     "react-stately": "^3.24.0",
-    "react-syntax-highlighter": "^15.5.0"
+    "react-syntax-highlighter": "^15.5.0",
+    "use-clipboard-copy": "^0.2.0"
```

Comment:
was going to use just the native browser clipboard function but this library was used in other component libraries and saw that it does provide nice ergonomics and handles some edge cases, and it's lightweight